src/main/scala/fi/oph/koski/valpas/ValpasOppijaSearchService.scala

```diff
@@ -158,6 +186,8 @@ case class ValpasLöytyiHenkilöhakuResult(
 }
 
 case class ValpasEiLainTaiMaksuttomuudenPiirissäHenkilöhakuResult(
+  oid: ValpasHenkilö.Oid,
```

Comment:
Hmm, mietin näitä hetu+oid:ia tässä, että palautetaanko ne nyt myös Valpas-kyselyn tuloksena frontille? Tällöin mahdollisesti frontin kautta saisi tiedon hetu-oid -yhdistelmästä myös oppijoista, joiden tietoihin käyttäjällä ei ole  mitään käyttöoikeutta. Varmaan melko harmiton tieto, mutta silti ylimääräistä.

Reply:
Voisin varmuuden vuoksi siivota ne pois.